contrib/cli/great_expectations_contrib/cli.py

```diff
@@ -39,16 +39,20 @@ def init() -> None:
 @click.pass_obj
 def publish(pkg: GreatExpectationsContribPackageManifest) -> None:
     publish_cmd()
-    pkg.update_package_state()
-    write_package_to_disk(pkg, PACKAGE_PATH)
+    sync_package(pkg, PACKAGE_PATH)
 
 
 @cli.command(help="Check your package to make sure it's met all the requirements")
 @click.pass_obj
 def check(pkg: GreatExpectationsContribPackageManifest) -> None:
     check_cmd()
-    pkg.update_package_state()
-    write_package_to_disk(pkg, PACKAGE_PATH)
+    sync_package(pkg, PACKAGE_PATH)
+
+
+@cli.command(help="Manually sync your package state")
+@click.pass_obj
+def sync(pkg: GreatExpectationsContribPackageManifest) -> None:
+    sync_package(pkg, PACKAGE_PATH)
```

Comment:
This is the hook that is called after each CLI invocation to update the underlying JSON object. I've exposed the functionality in a command to aid with debugging.